ci-operator/step-registry/opendatahub-io/ci/image-mirror/opendatahub-io-ci-image-mirror-commands.sh

```diff
@@ -47,6 +47,34 @@ if [[ -z "$IMAGE_TAG" ]]; then
             ;;
     esac
 fi
+
+# Get IMAGE_TAG in YearIndex format
+if [[ "$IMAGE_TAG" == "YearIndex" ]]; then
```

Comment:
Can you add a quick comment to explain the YearIndex format?  Just provide a quick example to show the `YearIndex` will be in the format of YYYYq_YYYMMDD and explain the quarter output.  This would make it easy for anyone learing the script to see what the format is without having to run the command locally

Reply:
Done! I updated also the references file `opendatahub-io-ci-image-mirror-ref.yaml` to explain the format of the date.